wordpress-mobile/WordPress-iOS
# Quick Start: add the post-creation alert

Refs: #9447 

This PR adds the fancy alert for Quick Start. This will allow users to opt-in to the Quick Start checklist.

![qs fancy alert](https://user-images.githubusercontent.com/517257/45646421-1ec6ff80-ba78-11e8-86ab-13a238592bc6.jpg)

**Notes**:
I've broken out some related changes to keep this PR small and focussed:
- the "Never" button has been omitted, to some in a PR that refactors `FancyAlertViewController` itself a little bit
- the storage of the opt-in value will come soon in a separate PR
- The changes in `BlogDetailsViewController+FancyAlerts.swift` were necessary to stop the Prime for Push alert from conflicting with this one.

To test:
- create a new site
- from the epilogue tap "configure" and ensure the new alert appears
- same for when tapping "write first post"
